packages/flutter/test/material/input_chip_test.dart

```diff
@@ -408,7 +408,7 @@ void main() {
     // Test default icon theme.
     await tester.pumpWidget(buildChip());
 
-    expect(getIconData(tester).color, ThemeData().iconTheme.color);
+    expect(getIconData(tester).color, ThemeData().colorScheme.onSurfaceVariant);
```

Comment:
Please add `theme` variable in this test and pass it to the `MaterialApp`. This ensures we're testing the expected `ThemeData` object.

```dart
    final ThemeData theme = ThemeData();

    Widget buildChip({ IconThemeData? iconTheme }) {
      return MaterialApp(
        theme: theme,
```

```suggestion
    expect(getIconData(tester).color, theme.colorScheme.onSurfaceVariant);
```